docs/rancher/csi-driver-support-various-sc.md

```diff
@@ -0,0 +1,84 @@
+---
+sidebar_position: 6
+sidebar_label: Various StorageClass of Harvester CSI Driver 
+title: "Various StorageClass of Harvester CSI Driver"
+keywords:
+  - Harvester
+  - harvester
+  - Rancher Integration
+  - CSI Driver
+  - StorageClass
+---
+
+After the Harvester CSI driver v0.1.15, you can create PVC based on the different StorageClass of Harvester.
+
+## Preparation
+
+It would be best if you did some preparation to support this feature entirely.
+
+1. Create a new `clusterrole` named `harvesterhci.io:csi-driver` with the following manifest.
+```
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  labels:
+    app.kubernetes.io/component: apiserver
+    app.kubernetes.io/name: harvester
+    app.kubernetes.io/part-of: harvester
+  name: harvesterhci.io:csi-driver
+rules:
+- apiGroups:
+  - storage.k8s.io
+  resources:
+  - storageclasses
+  verbs:
+  - get
+  - list
+  - watch
+```
+
+1. Then create the `clusterrolebinding` to associate with this. You can refer to the following manifest.
+```
+  apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: <namespace>-<serviceaccount name>
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: harvesterhci.io:csi-driver
+subjects:
+- kind: ServiceAccount
+  name: <serviceaccount name>
+  namespace: <namespace>
+  ```
+
+:::note
+
+Please complete these preparations on the Harvester Cluster.
+
+Also, the `serviceaccount name` and `namespace` should be the same as your could provider.
```

Comment:
can u please provide a guideline for how users could allocate them accordingly?

Reply:
Updated! Please help to review the latest version! Thanks